MOBILE_API.xml

```diff
@@ -1688,56 +1688,104 @@
+        <param name="utcMonth" type="Integer" minvalue="1" maxvalue="12" mandatory="false" since="5.0">
             <description>The current UTC month.</description>
+            <history>
+                <param name="utcMonth" type="Integer" minvalue="1" maxvalue="12" mandatory="true" since="2.0" until="5.0"/>
+            </history>
         </param>
-        <param name="utcDay" type="Integer" minvalue="1" maxvalue="31" mandatory="true">
+        <param name="utcDay" type="Integer" minvalue="1" maxvalue="31" mandatory="false" since="5.0">
             <description>The current UTC day.</description>
+            <history>
+                <param name="utcDay" type="Integer" minvalue="1" maxvalue="31" mandatory="true" since="2.0" until="5.0"/>
+            </history>
         </param>
-        <param name="utcHours" type="Integer" minvalue="0" maxvalue="23" mandatory="true">
+        <param name="utcHours" type="Integer" minvalue="0" maxvalue="23" mandatory="false" since="5.0">
             <description>The current UTC hour.</description>
+            <history>
+                <param name="utcHours" type="Integer" minvalue="0" maxvalue="23" mandatory="true" since="2.0" until="5.0"/>
+            </history>
         </param>
-        <param name="utcMinutes" type="Integer" minvalue="0" maxvalue="59" mandatory="true">
+        <param name="utcMinutes" type="Integer" minvalue="0" maxvalue="59" mandatory="false" since="5.0">
             <description>The current UTC minute.</description>
+            <history>
+                <param name="utcMinutes" type="Integer" minvalue="0" maxvalue="59" mandatory="true" since="2.0" until="5.0"/>
+            </history>
         </param>
-        <param name="utcSeconds" type="Integer" minvalue="0" maxvalue="59" mandatory="true">
+        <param name="utcSeconds" type="Integer" minvalue="0" maxvalue="59" mandatory="false" since="5.0">
             <description>The current UTC second.</description>
+            <history>
+                <param name="utcSeconds" type="Integer" minvalue="0" maxvalue="59" mandatory="true" since="2.0" until="5.0"/>
+            </history>
         </param>
-        <param name="compassDirection" type="CompassDirection" mandatory="true">
+        <param name="compassDirection" type="CompassDirection" mandatory="false" since="5.0">
             <description>See CompassDirection.</description>
+            <history>
+                <param name="compassDirection" type="CompassDirection" mandatory="true" since="2.0" until="5.0"/>
+            </history>
         </param>
-        <param name="pdop" type="Float" minvalue="0" maxvalue="10" defvalue="0" mandatory="true">
-            <description>PDOP.  If undefined or unavailable, then value shall be set to 0.</description>
+        <param name="pdop" type="Float" minvalue="0" maxvalue="1000" mandatory="false" since="5.0">
+            <description>PDOP.</description>
```

Comment:
Per the proposal, this description should be `PDOP.  If undefined or unavailable, then value shall be set to 0.`